packages/template-retail-react-app/app/pages/account/order-detail.jsx

```diff
@@ -55,7 +90,7 @@ const AccountOrderDetail = () => {
                 <Box>
                     <Button
                         as={Link}
-                        to={`${url.replace(`/${params.orderNo}`, '')}`}
+                        to={`${url?.replace(`/${params.orderNo}`, '')}`}
```

Comment:
In which case url is undefined? . What changes that cause the url to be falsy compared to previou implementation?  🤔

Reply:
No changes, just adding a guard as best practice.

The link was already broken in production https://pwa-kit.mobify-storefront.com/.

I changed the logic to directly the order history url.
The `<Link />` component already uses `buildUrl()` to add the localization proper paths to the url.